arches/app/models/graph.py

```diff
@@ -118,6 +117,12 @@ def __init__(self, *args, **kwargs):
                 for card in args[0]["cards"]:
                     self.add_card(card)
 
+                if "spatial_views" in args[0]:
```

Comment:
this probably needs to be added to the block on line 87

Reply:
@chrabyrd I didn't really understand that; can you explain why you're hanging stuff off the graph object there?

Reply:
discussed this; done now.